main/client/features/lifespanChart/lifespanChartDirective/lifespanChartDir.js

```diff
@@ -0,0 +1,101 @@
+angular.module('app').directive('lifespanChart', ($parse, $window) => {
+
+    return {
+		restrict: 'EA'
+        , templateUrl: './dist/html/lifespanChart/lifespanChartDirective/lifespanChartTemp.html'
+        , controller: 'lifespanChartCtrl'
+        , link(scope, elem, attrs) {
+
+	
+			var height = 500;
+			var width = 800;
+			var padding = 20;
+			var svg = d3.select(".chart")
+				.append("svg")
+				.attr({
+					height: height
+					, width: width
+				});
+				
+			var xscale = d3.scale.linear()
+				.domain([d3.min(scope.dataset, function(data) {
+					return data[0];}), 
+				d3.max(scope.dataset, function (data) {
+					return data[0];
+				})])
+				.range([padding, width-padding]);
+
+			var yscale = d3.scale.linear()
+				.domain([d3.min(scope.dataset, function(data) {
+					return data[0];}), 
+				d3.max(scope.dataset, function (data) {
+					return data[1];
+				})])
+				.range([height - padding, padding]);
+
+			var rscale = d3.scale.linear()
+				.domain([d3.min(scope.dataset, function(data) {
+					return data[0];}), 
+				d3.max(scope.dataset, function (data) {
+					return data[2];
+				})])
+				.range([10, 50]);
+
+
+			// var brush = d3.svg.brush()
+			// 	.x(xscale)
+			// 	.extent(d3.min(scope.dataset, function(data) {
+			// 		return data[0];}), 
+			// 	d3.max(scope.dataset, function (data) {
+			// 		return data[1];
+			// 	})])
+				
+				
+			svg.append("rect")
+				.attr('width', "100%")
```

Comment:
Try to stay consistent in how attributes are written. Currently you are using object notation in some places and single attr's in others.